lib/console/schema/deployment_settings.ex

```diff
@@ -21,6 +28,15 @@ defmodule Console.Schema.DeploymentSettings do
       |> cast(attrs, ~w(host user password)a)
       |> validate_required([:host])
     end
+
+    def to_headers(%__MODULE__{user: u, password: p}) when is_binary(u) and is_binary(p) do
```

Comment:
i'd just name these functions `headers` and `url` (the to is kind of redundant and rather save the characters)